src/dbt_osmosis/main.py

```diff
@@ -434,6 +440,7 @@ def document(
     add_progenitor_to_meta: bool = False,
     profile: Optional[str] = None,
     vars: Optional[str] = None,
+    use_direct_yaml_descriptions: bool = False,
```

Comment:
Maybe use_unrendered_descriptions is better. We would also add this to the `refactor` command which is a superset of document and organize, so we can push the kwarg down there too.